src/ManagedArray.hpp

```diff
@@ -56,6 +56,18 @@ namespace chai {
 
 struct InvalidConstCast;
 
+/*!
+ * \class CHAICopyable
+ *
+ * \brief Provides an interface for types that are copyable.
+ *
+ * If a class inherits from CHAICopyable, then a data transfer is triggered
```

Comment:
Might want to clarify that it triggers a copy each of the stored items of type `T`.

Reply:
Done.